CMakeLists.txt

```diff
@@ -630,6 +630,26 @@ install(TARGETS isle ${install_extra_targets}
   RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
   LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
 )
+if (ISLE_BUILD_CONFIG)
+  find_program(WINDEPLOYQT_EXECUTABLE windeployqt)
+  if(WINDEPLOYQT_EXECUTABLE)
+    add_custom_command(TARGET config POST_BUILD
+      COMMAND "${WINDEPLOYQT_EXECUTABLE}"
+              "$<TARGET_FILE:config>"
+              "--no-compiler-runtime"
+              "--no-opengl-sw"
+              "--no-system-d3d-compiler"
+              "--no-translations"
+              "--no-quick-import"
+      COMMENT "Running windeployqt with minimal dependencies"
+    )
```

Comment:
I think you might want to do this at install time, using [`install(CODE)`](https://cmake.org/cmake/help/latest/command/install.html#code).

Reply:
I was unable to get that working.